packages/sandbox/src/render/index.ts

```
@@ -21,7 +21,7 @@ export function createSandbox(
   customOptions: Partial<EjecutarOptions>,
 ): HTMLIFrameElement {
   if (typeof elementOrId === 'string') {
-    elementOrId = document.getElementById(elementOrId) as HTMLElement;
+    elementOrId = document.getElementById(elementOrId) as HTMLElement
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)